test/sqllogictest/transform/fold_constants.slt

```diff
@@ -143,3 +143,650 @@ Explained Query:
           Get l0 // { arity: 1 }
 
 EOF
+
```

Comment:
Shall I move the following code to separate files to reduce the length of this one?

Reply:
=> I split them by computation mode (to allow easier comparisons) and content (to reduce the file length).